Automattic/sensei
# Fix issue with relative paths on SCSS files on Sensei LMS

Alternative to #5929

### Changes proposed in this Pull Request

* Install `resolve-url-loader` and use it to resolve relative paths in `url(...)` commands, in SCSS files;

### Testing instructions

1. Run `npm start`, and verify if it works correctly;
2. Create a build using `npm run build`, and verify if the created build works correctly;

### Context

This PR fixes an issue where relative paths inside SCSS files were not working correctly, in special when importing files from Sensei Pro, or using relative paths inside SCSS files imported by other SCSS files.

This PR fixes this issue by using the instructions [provided here](https://github.com/webpack-contrib/sass-loader#resolving-import-at-rules), and fixes the issue with the `url(...)` commands by [following these instructions](https://github.com/webpack-contrib/sass-loader#problems-with-url).
